smoke-tests/smoke-agent-manual.bats

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bats
+
+load test_helpers/utilities
+
+setup_file() {
+	echo "# 🚧" >&3
+	echo -n "# 🍿 Setting up smoke-agent-manual ..." >&3
+	docker-compose up --detach collector app-agent-manual
+	until [[ $(docker-compose logs app-agent-manual | grep "OK I'm ready now") ]]; do sleep 1; done
```

Comment:
I re-remember that this wait loop in the before_all of each app variant smoke test does not have a timeout. 🤔